carousel.js

```diff
@@ -0,0 +1,381 @@
+'use strict';
```

Comment:
`carousel.js` and `indicator-dots.js` shouldn't be included, they are only built for release to npm.
I should refine the build process, avoid generate temporary files in root dir. 🤔